source/MaterialXTest/CMakeLists.txt

```diff
@@ -36,6 +36,18 @@ file(GLOB materialx_headers "${CMAKE_CURRENT_SOURCE_DIR}/*.h")
 
 list(APPEND materialx_headers "${CMAKE_CURRENT_SOURCE_DIR}/Catch/catch.hpp")
 
+if (NOT MATERIALX_BUILD_GEN_ARNOLD)
```

Comment:
This should not be restored. There is a scan for each directory now to find files to include (add_tests above).